autopush/web/webpush.py

```diff
@@ -54,6 +54,8 @@
 # Base64 URL validation
 VALID_BASE64_URL = re.compile(r'^[0-9A-Za-z\-_]+=*$')
 
+VALID_ROUTER_TYPES = ["simplepush", "webpush", "gcm", "fcm", "apns"]
```

Comment:
Really thought I had something like this before. There's a similar list in `autopush.registration.conditional_token_check` Wonder if it might make sense to have this list somewhere common and use it in both locations?

Reply:
Yea, or heck, maybe a constants.py for the shared universal constants we end up using all over.

Reply:
absolutely 👍 to a constants.py. I've got one for boto I put in `__init__.py`, but happy to move it in favor of just keeping `version` there.

Reply:
Will open a separate PR to refactor some of the constants together.